Solution for Library Task without ConsoleProgram

RealWhimsy opened the pull request:
Modified the existing solution to work without ConsoleProgram. Now uses BufferedReader to read user inputs directly from the IDE's console. Wrong user inputs are caught and reacted to by re-reading the input. Comments should make the code understandable even if some principles like try-catch have not been explained in the lecture yet.

RealWhimsy committed:
Added solution for Task 1 (Library)

RealWhimsy committed:
Made Program work without ConsoleProgram

The program now uses the BufferedReader class to read User Input in the IDE's built in console and no longer uses ConsoleProgram as a super class. Input mismatches are caught using try-catch. Should the user enter a wrong input type, eg. "zweitausend" for 'year', a error message is displayed and the method is restarted, so that the user can provide a new input.

RealWhimsy committed:
Added comments

RealWhimsy committed:
Added more comments

alexanderbazo commented:
[In der Vorlesung](https://regensburger-forscher.de/oop-slides/index.html?slides=06-Klassenmodellierung-1-Grundlagen&startAtSlide=9) habe ich die Scanner-Klasse für die Eingabe verwendet. Damit sind auch `int` oder `boolean`-Werte ohne *Parsing* einlesbar. Kann man das noch ändern?

RealWhimsy committed:
Replaced BufferedReader with Scanner

Now uses the Scanner class for reading user input instead of BufferedReader. Changed code and edited comments accordingly

RealWhimsy commented:
Benutzt nun nach Commit 217fdc7b4dd1a751d6477624e2943d0e5c9422bd die Scanner - Klasse zum einlesen von User Input
